packages/scanner/src/types.d.ts

```diff
@@ -111,6 +114,20 @@ interface Finding {
   groupMessage?: string;
   occurranceCount?: number;
   relatedEvents?: Event[];
+  // NOTE: These fields are experimental at this time.
+  // An extended description of the finding. The text of this description can use Markdown, plus an
+  // extended syntax.
+  //
+  // Examples:
+  // {value} Locate the indicated key (value) in volatileData, and format as text.
+  // {value as regexp} Locate the indicated key (value) in volatileData, and format as a regexp.
+  // {logEvent as url} Locate the indicated key (logEvent) in participatingEvents, and format as a URL.
+  // {logEvent as prettyName} Locate the indicated key (logEvent) in participatingEvents, and format
+  // as a prettified event or code object name.
+  description?: string;
+  // Map of events by functional role name; for example, logEvent, secret, scope, etc.
+  participatingEvents?: Record<string, Event>;
+  volatileData?: Record<string, string | number>;
```

Comment:
Perhaps these and other properties common with `MatchResult` should be refactored to a common base interface. It could maybe also be used for `buildFinding` argument (see the comment above).